STPullToRefresh/STPullToRefreshHelper.h

```diff
@@ -42,9 +44,13 @@ typedef enum STPullToRefreshState {
 - (id)initWithDirection:(STPullToRefreshDirection)direction delegate:(id<STPullToRefreshHelperDelegate>)delegate;
 - (id)initWithDirection:(STPullToRefreshDirection)direction viewClass:(Class)viewClass delegate:(id<STPullToRefreshHelperDelegate>)delegate;
 
+@property (nonatomic) BOOL shouldKeepPullToRefreshViewAtFront;
+@property (nonatomic) CGFloat pullToTriggerHeightMultiplier;
```

Comment:
Agreed. I prefer `triggerDistance`. Should be able to tweak the view height without changing the pull to refresh feel.

The lazily went with multiplier because 1.0 is a an easy default. I'll make it so that it uses the view height if `triggerDistance` is zero.